service/src/main/java/org/exoplatform/extension/exchange/service/util/CalendarConverterUtils.java

```diff
@@ -460,7 +463,7 @@ public static void convertExoToExchangeEvent(Appointment appointment,
         repeatInterval = 1;
       }
       if (repeatType.equals(CalendarEvent.RP_WEEKLY)) {
-        Utils.adaptRepeatRule(event, CalendarService.PERSISTED_TIMEZONE, DEFAULT_TIME_ZONE);
+        Utils.adaptRepeatRule(event, CalendarService.PERSISTED_TIMEZONE, DateUtils.getTimeZone(TimeUtil.TIMEZONE_MAPPINGS.get(DEFAULT_TIME_ZONE_EXCHANGE)));

```

Comment:
Same comment as previous for multiple lines instruction

Reply:
Fixed.